src/components/UserSearch.js

```diff
@@ -48,7 +46,7 @@ class UserSearch extends Component {
 
     await foundUsers.forEach(async foundUser => {
       const userDetails = await fetch(
-        `https://pub.orcid.org/v2.1/${foundUser["orcid-identifier"].path}/personal-details`,
+        `https://pub.orcid.org/v2.1/${foundUser["orcid-identifier"].path}/person`,
```

Comment:
This is a very nice way to get more personal details since we don't need to store them (GDPR) and don't need to give public access to the internally stored email address.